direct/src/extensions_native/NodePath_extensions.py

```diff
@@ -10,309 +11,330 @@
-        """ Deprecated.  Pretty print a node path's pos and, hpr """
-        print("NodePath.printPosHpr() is deprecated.")
-        formatString = '%0.' + '%d' % sd + 'f'
-        if other:
-            pos = self.getPos(other)
-            hpr = self.getHpr(other)
-            otherString = other.getName() + ', '
-        else:
-            pos = self.getPos()
-            hpr = self.getHpr()
-            otherString = ''
-        print((self.getName() + '.setPosHpr(' + otherString +
-               formatString % pos[0] + ', ' +
-               formatString % pos[1] + ', ' +
-               formatString % pos[2] + ', ' +
-               formatString % hpr[0] + ', ' +
-               formatString % hpr[1] + ', ' +
-               formatString % hpr[2] +
-               ')\n'))
+    """ Deprecated.  Pretty print a node path's pos and, hpr """
+    if __debug__:
+            warnings.warn("NodePath.printPosHpr() is deprecated.", DeprecationWarning, stacklevel=2)
+    formatString = '%0.' + '%d' % sd + 'f'
+    if other:
+        pos = self.getPos(other)
+        hpr = self.getHpr(other)
+        otherString = other.getName() + ', '
+    else:
+        pos = self.getPos()
+        hpr = self.getHpr()
+        otherString = ''
+    print((self.getName() + '.setPosHpr(' + otherString +
+           formatString % pos[0] + ', ' +
+           formatString % pos[1] + ', ' +
+           formatString % pos[2] + ', ' +
+           formatString % hpr[0] + ', ' +
+           formatString % hpr[1] + ', ' +
+           formatString % hpr[2] +
+           ')\n'))
 
 Dtool_funcToMethod(printPosHpr, NodePath)
 del printPosHpr
 #####################################################################
 def printPosHprScale(self, other = None, sd = 2):
-        """ Deprecated.  Pretty print a node path's pos, hpr, and scale """
-        print("NodePath.printPosHprScale() is deprecated.")
-        formatString = '%0.' + '%d' % sd + 'f'
-        if other:
-            pos = self.getPos(other)
-            hpr = self.getHpr(other)
-            scale = self.getScale(other)
-            otherString = other.getName() + ', '
-        else:
-            pos = self.getPos()
-            hpr = self.getHpr()
-            scale = self.getScale()
-            otherString = ''
-        print((self.getName() + '.setPosHprScale(' + otherString +
-               formatString % pos[0] + ', ' +
-               formatString % pos[1] + ', ' +
-               formatString % pos[2] + ', ' +
-               formatString % hpr[0] + ', ' +
-               formatString % hpr[1] + ', ' +
-               formatString % hpr[2] + ', ' +
-               formatString % scale[0] + ', ' +
-               formatString % scale[1] + ', ' +
-               formatString % scale[2] +
-               ')\n'))
+    """ Deprecated.  Pretty print a node path's pos, hpr, and scale """
+    if __debug__:
+            warnings.warn("NodePath.printPosHprScale() is deprecated.", DeprecationWarning, stacklevel=2)
```

Comment:
This line is indented too much, and there are several more like that in this file.

Reply:
Indents removed. This should be the last of that.